docs/mevblocker/README.mdx

```diff
@@ -1,4 +1,40 @@
----
-title: MEVBlocker
-draft: true
----
+# MEV Blocker
+
+A product developed and stewarded by CoW DAO, Beaver Builder and Gnosis DAO.
+
+IMG
```

Comment:
Please include the image

Reply:
Well, I havent finished it yet, was just testing it